R/input-slider.R

```diff
@@ -90,6 +90,15 @@ sliderInput <- function(inputId, label, min, max, value, step = NULL,
                     version = "0.10.2.2")
   }
 
+  if (value < min | value > max) {
```

Comment:
I think this needs to handle the case where `value` has two values.

Reply:
Done via last commit ✅